Modules/_collectionsmodule.c

```diff
@@ -1102,7 +1102,7 @@ deque_index(dequeobject *deque, PyObject *const *args, Py_ssize_t nargs)
 }
 
 PyDoc_STRVAR(index_doc,
-"index(x, [start, [stop]]) -> int\n\n"
+"index(x, [start, [stop]]) -> int\n"
 "Return the position of *x* in the deque "
 "(at or after index *start* and before index *stop*). "
 "Returns the first match or raises a ValueError if not found.");
```

Comment:
Ok, the reason I changed this was to make it consistent to the [docs](https://github.com/python/cpython/blob/main/Doc/library/collections.rst), but I reverted the wording now. Are you fine with replacing `value` by `x` to make it consistent with the function signatures of the other methods?

At some time in the future we could also think about using Sphinx' autodoc to use the docstrings in the documentation so we don't have to write everything twice and have potential inconsistencies... Do you think that's worth another issue, or was this debated and rejected already?

Reply:
There has been discussion about it, yeah, and in an ideal world it would be nice to have, but it would be a very complex and involved undertaking to begin with, and that aside, a fundamental issue remains that (as you can see from the brouhaha on this PR) there are different user needs, requirements and conventions for docstrings in the code as opposed to prose documentation in separate documents. For now, at least, the most practical approach seems to be to simply proceed with improving them separately for now.